client/app/views/steps/password.coffee

```diff
@@ -1,14 +1,27 @@
 StepView = require '../step'
+timezones = require '../../lib/timezones'
+
 
 module.exports = class PasswordView extends StepView
+
     template: require '../templates/view_steps_password'
 
     events:
-        'click button': 'onSubmit'
+        'click button': 'doSubmit'
+
+
+    getDataFromDOM: ->
+        # TODO: get user info
+        # from server
+        return {
+            email: 'toto@cozycloud.cc'
+            public_name: 'toto'
+            timezone: timezones[0]
+            password: @$('input[name=password]').val()
+            allow_stats: false
+        }
```

Comment:
@CPatchane : Sorry, I misunderstood :( I will remove useless params; but anyway I don't understand what `onboardedSteps` is for? It actually works without. Could you explain that to me please?